graphql/schema.graphql

```diff
@@ -1,6 +1,12 @@
 scalar JSON
 scalar JSONObject
 
+type CorpDocData {
```

Comment:
Just making explicit the "documentation" suggestion that I just mentioned:

```suggestion
// https://www.oecd.org/tax/automatic-exchange/crs-implementation-and-assistance/tax-identification-numbers/
type LegalPersonTaxId {
```

or 

```suggestion
// https://www.oecd.org/tax/automatic-exchange/crs-implementation-and-assistance/tax-identification-numbers/
type TaxId {
```